app/views/products/index.html.haml

```diff
@@ -0,0 +1,12 @@
+%nav.navbar.navbar-default{:role => "navigation"}
+      %ul.nav.navbar-nav
+      %li
+      %a{:href => "#"} Cart
```

Comment:
We do not want to use hash rockets, old syntax

e.g
`%a{ href: "#" } Cart`

Reply:
Done!